CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ _This release is scheduled to be released on 2021-04-01._
 ### Removed
 
 - Removed danger.js library.
+- Removed ical.
```

Comment:
bigger maybe :-) ? seriously: isnt there some deprecation / warning mechanism for module developers?

Reply:
I don't know about such a deprecation / warning mechanism, would this be sufficient for the changelog?
- Removed `ical` which was substituted by `node-ical` in release `v2.13.0`. Module developers must install this dependency themselves in the module folder if needed.